index.html

```diff
@@ -1389,6 +1381,82 @@
       <a>Web NFC message</a> can be set for sending.
     </p>
   </section>
+  <section> <h3>The <strong>NFCWatchOptions</strong> dictionary</h3>
+      <p>
+        To describe which messages an application is interested in, the
+        <dfn>NFCWatchOptions</dfn> dictionary is used:
+      </p>
+      <pre class="idl">
+        enum NFCWatchMode {
+          "web-nfc-only",
+          "any"
+        };
+
+        dictionary NFCWatchOptions {
+          USVString url = null;
```

Comment:
I think the convention for dictionary fields is to leave them `undefined` when they're not present, if the only other default would be `null`. I'm not sure of that, so no need to change it, but this may come up later.

I am pretty sure that if `null` is an allowed value, the type needs to be [`USVString?`](https://heycam.github.io/webidl/#idl-nullable-type).